mtv_dl.py

```diff
@@ -39,6 +39,8 @@
 
 List options:
   -c <results>, --count=<results>       Limit the number of results. [default: 50]
+  --series                              Try to extract season and episode information from the title
```

Comment:
Do you want me to strip the `series_mode` flag and all the if-statements in the `show_table` function as well? Or should we leave them in, just in case?

Reply:
And yes, this worked for me...

Reply:
For me it looks like this:

```
> poetry run mtv_dl list --series
--series is not a unique prefix: --series, --series?
Usage:
  mtv_dl.py list [options] [--sets=<file>] [--count=<results>] [<filter>...]
  mtv_dl.py dump [options] [--sets=<file>] [<filter>...]
  mtv_dl.py download [options] [--sets=<file>] [--low|--high] [<filter>...]
  mtv_dl.py history [options] [--reset|--remove=<hash>]
  mtv_dl.py --help
```

… yes i would also remove the if-statements there to be in line with the docs.